index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <div class="wrapper">
+            <header class="header-section">
+                <nav class="logo"><a href="#">My Logo</a></nav>
+                <nav><img src="/images/Enabled.png" alt="enable image"></nav>
```

Comment:
- [ ] Kindly as requested by the previous reviewer, the semantic help the browser to know witch section it gonna display as well. instead of having two `nav` you can use `div` like: `<div><a>.....</a></div>` for the logo and  `<div><a>.....</a></div>` fr the image but both inside a `nav` as a parent